guides/rails_guides/markdown/renderer.rb

```diff
@@ -90,6 +94,19 @@ def clipboard_content(code, language)
             code = code.lines.grep(prompt_regexp).join.gsub(prompt_regexp, "")
           end
 
+          # Remove comments that reference an application file.
+          filepath_regexp =
+            case language
+            when "erb", "html+erb"
+              /<%# #{APPLICATION_FILEPATH_REGEXP}.* %>/
+            when "ruby", "yaml", "yml"
+              /# #{APPLICATION_FILEPATH_REGEXP}/
```

Comment:
Should we check for start of line, just in case? I don't _think_ we'd find these anywhere else, but perhaps to be extra safe?

```suggestion
              /^<%# #{APPLICATION_FILEPATH_REGEXP}.* %>/
            when "ruby", "yaml", "yml"
              /^# #{APPLICATION_FILEPATH_REGEXP}/
```